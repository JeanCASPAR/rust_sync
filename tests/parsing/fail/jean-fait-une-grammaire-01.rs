use rust_sync::sync;

sync! {
    #[pass(0)]
    
    node oui() = ()
    where
        x : bool = !!true;
}

fn main() {}
