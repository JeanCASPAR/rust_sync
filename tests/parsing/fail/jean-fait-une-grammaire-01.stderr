error: expected one of: `if`, `merge`, parentheses, integer literal, floating point literal, boolean literal, identifier
 --> tests/parsing/fail/jean-fait-une-grammaire-01.rs:8:21
  |
8 |         x : bool = !!true;
  |                     ^
